### ISSUE TITLE
InsertText function error when use custom font

### ISSUE THREAD

**Issue by Samuraiwarm** [pymupdf/PyMuPDF#319]
# Code
```
page.insertFont(fontname='F0', set_simple=False, fontfile='/fonts/THSarabunNew.ttf')
page.insertText(
                    fitz.Point(100, 100),
                    'Hello World This is thai language: ทดสอบภาษาไทย',
                    fontname='F0',
                    fontsize=16,
                    color=(0,0,0),
                    overlay=True
                )
```

# Expected Behavior
PDF should be generated and render text correctly.

# Actual Behavior
I get error below

```
Exception :  list index out of range
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/fitz/utils.py", line 1120, in insertText
    morph=morph)
  File "/usr/local/lib/python3.6/site-packages/fitz/utils.py", line 2369, in insertText
    tab.append(getTJstr(t, g, simple, ordering))
  File "/usr/local/lib/python3.6/site-packages/fitz/fitz.py", line 1602, in getTJstr
    otxt = "".join([hex(glyphs[ord(c)][0])[2:].rjust(4, "0") for c in text])
  File "/usr/local/lib/python3.6/site-packages/fitz/fitz.py", line 1602, in <listcomp>
    otxt = "".join([hex(glyphs[ord(c)][0])[2:].rjust(4, "0") for c in text])
IndexError: list index out of range
```

# Font file
[THSarabunNew.zip](https://github.com/pymupdf/PyMuPDF/files/3368436/THSarabunNew.zip)

**Comment by JorjMcKie** [comment]
... investigating.
There definitely is an error in ``insertText``, no exception will happen if you use ``insertTextbox`` instead.
I don't have your specific font on my Windows system, but when using a MS supplied Thai font, no Thai letters are printed.
So this is a secondary issue which may or may not happen on your Linux. Please feel free to try it using `insertTextbox`.

**Comment by Samuraiwarm** [comment]
It works now with `insertTextbox`. Thank you!

**Comment by Samuraiwarm** [comment]
It works now with `insertTextbox`. Thank you!

**Comment by JorjMcKie** [comment]
thanks for the feedback.
I will fix the error in ``insertText`` then.

**Comment by Samuraiwarm** [comment]
For font file, I have already attach file in issue. You can download here: https://github.com/pymupdf/PyMuPDF/files/3368436/THSarabunNew.zip

**Comment by JorjMcKie** [comment]
Thank you. I found another (correct) Thai font in my Windows installation - obviously the first one just was something else.
Now it works.
Already fixed the bug in ``insertText``. In case you need to use this method urgently, I am sending you a zip of ``utils.py`` which contains the hotfix:
[utils.zip](https://github.com/pymupdf/PyMuPDF/files/3372116/utils.zip)
If you copy it over to your Python's ``fitz`` dir, the method should work.

**Comment by JorjMcKie** [comment]
Here is the test code snippet:
```python
>>> import fitz
>>> fontfile="c:/windows/fonts/LeelawUI.ttf"
>>> text = 'Hello World This is thai language: ทดสอบภาษาไทย'
>>> doc=fitz.open()
>>> page=doc.newPage()
>>> page.insertText((100,100), text, fontfile=fontfile, fontname="f0")
1
>>> doc.save("thai.pdf")
>>> 
```
and the resulting page:

![grafik](https://user-images.githubusercontent.com/8290722/60875116-3c33a880-a207-11e9-8d67-755959327850.png)

**Comment by Samuraiwarm** [comment]
Thank you for the fix!

**Comment by JorjMcKie** [comment]
closed with v1.14.18
